packages/docs/pages/Table/TablePage.tsx

```diff
@@ -172,6 +172,50 @@ const TablePage = () => {
         {/* jsx-to-string:end */}
       </CodePreview>
 
+      <H1>Usage with TableFigure</H1>
+
+      <Text>
+        TableFigure components are used to wrap Tables and any relevant information to be grouped with them.
+        TableFigures also provide a scrollable overflow on mobile for Tables with large amounts of data. Try removing
+        the TableFigure component below in mobile view to see the differences.
+      </Text>
+
+      <CodePreview>
+        {/* jsx-to-string:start */}
+        <>
+          <TableFigure>
+            <Table
+              columns={[
+                { header: 'Sku', hash: 'sku', render: ({ sku }) => sku },
+                { header: 'Name', hash: 'name', render: ({ name }) => name },
+                { header: 'Description', hash: 'description', render: ({ description }) => description },
+              ]}
+              items={[
+                {
+                  sku: 'SM13',
+                  name: 'Item 1',
+                  description:
```

Comment:
Can we use our actual SKU data here (`name` and `description`)?

Reply:
Sure - can I ask why it matters (for future reference)?